client/src/components/scheduled-events/DateRange/DateRange.vue

```diff
@@ -125,6 +125,16 @@ export default {
             } else {
                 this.loadMoreActive = false;
             }
+        },
+
+        getEndDateDateRange(endDate) {
+            let endDateFormatter = new Date(endDate);
+            endDateFormatter = endDateFormatter.getTime() + 86400000;
+            endDateFormatter = new Date(endDateFormatter)
+                .toISOString()
+                .substr(0, 10);
+
+            return endDateFormatter;
```

Comment:
There are much better ways of doing the same. We have `moment` for working with dates.
```
return moment.utc(endDate).endOf('day').format('YYYY-MM-DD');